packages/block-editor/src/components/block-list/block-wrapper.js

```diff
@@ -87,10 +90,13 @@ export function useBlockProps( props = {}, { __unstableIsHtml } = {} ) {
 					? getSelectedBlocksInitialCaretPosition()
 					: undefined,
 				isNavigationMode: _isNavigationMode,
+				isFullSiteEditingActive: getSettings()
+					.__experimentalEnableFullSiteEditing,
```

Comment:
since this feature is only used by FSE right now, we thought it might be wise to only enable it in FSE to avoid any possible performance regressions in the main plugin

Reply:
Did we check/notice performance issues?

Reply:
yeah, we did spend time looking at and fixing performance problems.

Reply:
So just to confirm, without this check we do have performance issues?

Reply:
What I'm saying is that performance doesn't seem to be a valid reason to add this check because otherwise FSE screen won't be performant anyway right?

Reply:
> So just to confirm, without this check we do have performance issues?

none that we have uncovered in the testing. We aren't gating this because of any performance issue we know of, just out of an abundance of caution

Reply:
I think personally we should remove it in that case, maybe add a comment explaining that it's only useful for FSE for now but i don't think we should be sacrificing puriness and architecture in favor of performance issues we're not sure exist.